main.go

```diff
@@ -15,28 +16,51 @@ import (
 	_ "github.com/jackc/pgx/v4/stdlib"
 	"go.uber.org/zap"
 	"golang.org/x/crypto/bcrypt"
+
+	"github.com/ilyakaznacheev/cleanenv"
 )
 
 // Main function
 func main() {
-	//fmt.Println("Test")
+	var env config.Env
+	err := cleanenv.ReadEnv(&env)
+	if err != nil {
+		log.Fatal(fmt.Errorf("read env: %w", err))
+	}
+
+	var config config.Config
+	err = cleanenv.ReadConfig("config.json", &config)
+	if err != nil {
+		log.Fatal(fmt.Errorf("read config: %w", err))
+	}
+
 	logger, _ := zap.NewProduction()
-	dsn := "postgres://postgres:secret@0.0.0.0:5432/test?sslmode=disable&timezone=utc"
-	db, err := sql.Open("pgx", dsn)
+
+	db, err := sql.Open(env.SqlDriver, env.Connection)
 	if err != nil {
 		log.Fatal(fmt.Errorf("creating db: %w", err))
 	}
 	err = db.Ping()
 	if err != nil {
 		db.Close()
-		log.Fatal(fmt.Errorf("ping db %s : %w", dsn, err))
+		log.Fatal(fmt.Errorf("ping db %s : %w", env.Connection, err))
 	}
 
 	migrationsPath := "migrations"
-	err = postgres.MigrateUp(migrationsPath, dsn)
+	err = postgres.MigrateUp(migrationsPath, env.Connection)
 	if err != nil {
 		log.Fatal(fmt.Errorf("db migrations: %w", err))
 	}
+
+	initTestData(db)
+
+	logger.Info("starting web server")
+	r := router.NewRouter(db, logger)
+	// Start server
+	log.Fatal(http.ListenAndServe(env.Host+":"+env.Port, r))
+}
+
+func initTestData(db *sql.DB) {
```

Comment:
I do not think we need it, maybe find a better place  then main, I done it just temprary